clients_tests is unstable

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

The clients_tests is now failing in main randomly.

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

The clients tests pass

### Actual behavior

The `Realm client` test in  clients_tests fails with

`AssertionError: Timed out retrying after 30000ms: Too many elements found. Found '6', expected '5'.`

### How to Reproduce?

Uncertain: Run the client tests preceded by some of the realm tests.

### Anything else?

The extra tab is the `Authorization` tab which happens to be sometimes visible with the `master-realm` client.
The workaround is to use a fresh realm client rather than `master-realm` which seems to enable authorization for this "client" at earlier stage but cannot disable it afterwards.

--- a/js/apps/admin-ui/cypress/e2e/clients_test.spec.ts
+++ b/js/apps/admin-ui/cypress/e2e/clients_test.spec.ts
@@ -1152,11 +1152,12 @@ describe("Clients test", () => {
   });
 
   describe("Realm client", () => {
-    const clientName = "master-realm";
+    const clientName = `${realmName}-realm`;
 
     beforeEach(() => {
       loginPage.logIn();
       keycloakBefore();
+      // Stay in master realm, do not switch to ${realmName} realm
       commonPage.sidebar().goToClients();
       commonPage.tableToolbarUtils().searchItem(clientName);
       commonPage.tableUtils().clickRowItemLink(clientName);

--- a/js/apps/admin-ui/cypress/support/pages/admin-ui/components/TablePage.ts
+++ b/js/apps/admin-ui/cypress/support/pages/admin-ui/components/TablePage.ts
@@ -39,7 +39,7 @@ export default class TablePage extends CommonElements {
         this.#tableRowItem,
     )
       .contains(itemName)
-      .click();
+      .click({ force: true });
     return this;
   }
 

